### PR TITLE
Do  not install plugin tests

### PR DESCRIPTION
**Pull request by lo-ca**

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/python-language-server](https://github.com/palantir/python-language-server), @lo-ca! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
-    packages=find_packages(exclude=['contrib', 'docs', 'test']),
+    packages=find_packages(exclude=['contrib', 'docs', 'test', 'test.plugins']),
```
Comment (gatesn) [inline setup.py:28]: Can you change this to `test.*`?

**Review by gatesn — CHANGES_REQUESTED** [review]
Thank you, this is a good catch! Let's wildcard everything though

**Comment by gatesn** [comment]
Closed in #493

**Pull request closed without merging**